TranslatorReasonerAPI.yaml

```diff
@@ -295,18 +295,26 @@ components:
     KnowledgeGraph:
       type: object
       description: >-
-        A thought graph associated with this result. This will commonly be a
-        linear path subgraph from one concept to another, but related items
-        aside of the path may be included.
+        A thought graph associated with this result. This will commonly be
```

Comment:
I think we discussed avoiding this "thought graph" terminology. Would it be out-of-line to use "knowledge graph" in the description here? Also, the use of "result" here is a bit weird because the Message contains a list of "results". How about: "The knowledge graph associated with a set of results."

Reply:
sure. np.